### ISSUE TITLE
<class 'KeyError'>: 'indices' when running various tasks 

### ISSUE THREAD

**Issue by zzztimbo** [elastic/curator#1551]
curator, version 5.8.1

curator is able to connect to my elastic cluster and read indices from it.

```
...
2020-07-30 16:36:00,458 DEBUG          curator.indexlist     __build_index_info:80   Building preliminary index metadata for svc-docker-registry-2020.07.10
2020-07-30 16:36:00,458 DEBUG          curator.indexlist     __build_index_info:80   Building preliminary index metadata for svc-autoscaler-2020.07.09
2020-07-30 16:36:00,458 DEBUG          curator.indexlist     __build_index_info:80   Building preliminary index metadata for svc-vouch-2020
2020-07-30 16:36:00,458 DEBUG          curator.indexlist          _get_metadata:177  Getting index metadata
2020-07-30 16:36:00,458 DEBUG          curator.indexlist       empty_list_check:226  Checking for empty list
2020-07-30 16:36:35,875 DEBUG          curator.indexlist       _get_index_stats:117  Getting index stats
2020-07-30 16:36:35,875 DEBUG          curator.indexlist       empty_list_check:226  Checking for empty list
2020-07-30 16:36:35,875 DEBUG          curator.indexlist           working_list:237  Generating working list of indices
2020-07-30 16:36:35,875 DEBUG          curator.indexlist           working_list:237  Generating working list of indices
2020-07-30 16:37:05,884 ERROR                curator.cli                    run:213  Failed to complete action: delete_indices.  <class 'KeyError'>: 'indices'
```

Sometimes it works, but often I see the error above. 
Would load on the cluster have something to do with this error?

**Comment by luliwater** [comment]
I also find the same problem occasionally when using  curator 5.8.1  with using action forcemerge:
`  5:
    action: forcemerge
    description: >-
      forcemerge index over a month to specified segments per shard
    options:
      max_num_segments: ${max_num_segments:1}
      timeout_override: 21600
      ignore_empty_list: True
    filters:
    - filtertype: pattern
      kind: prefix
      value: ${match_pattern}
    - filtertype: allocated
      key: temperature
      value: warm
      allocation_type: require
      exclude: False
    - filtertype: forcemerged
      max_num_segments: ${max_num_segments:1}
      exclude: True`

 
Can someone help us?   

[IsOkay  debug level log](https://github.com/elastic/curator/files/5614586/IsOkay.debug.level.log)
[KeyError debug level log](https://github.com/elastic/curator/files/5614587/KeyError.debug.level.log)

**Comment by untergeek** [comment]
Probably better addressed in #1670
